File: Backend/apptienda/src/main/java/com/iesagora/jesus/apptienda/business/dto/PedidoDTO.java
package com.iesagora.jesus.apptienda.business.dto;

import java.math.BigDecimal;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class PedidoDTO {
	
	private List<DetallePedidoDTO> detallePedidoDTO;
	private BigDecimal total;
	private String direccionCliente;
	private String cpCliente;
	private String ciudadCliente;
	private String provinciaCliente;
	private String paisCliente;

}

File: Frontend/apptienda/src/app/model/pedido.dto.ts
import { DetallePedidoDTO } from "./detalle-pedido.dto";

export interface PedidoDTO {
    detallePedidoDTO: DetallePedidoDTO[];
    total: number;
    direccionCliente: string;
    cpCliente: string;
    ciudadCliente: string;
    provinciaCliente: string;
    paisCliente: string;
}